wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/WbEditingAction.java

```diff
@@ -736,7 +739,9 @@ private JsonNode performAPIAction(
 			return null;
 		}
 
-		checkEditSpeed();
+		if (!bot) {
+			checkEditSpeed();
+		}
```

Comment:
The edit speed should be controlled even for bots. If you want to disable it, just configure the `WikibaseDataEditor` for a high editing speed: http://wikidata.github.io/Wikidata-Toolkit/org/wikidata/wdtk/wikibaseapi/WikibaseDataEditor.html#setAverageTimePerEdit(int)

Reply:
Thanks, it's working.